index.html

```diff
@@ -63,12 +63,13 @@
         <span class="icon-bar"></span>                        
       </button>
       <a class="navbar-brand" href="#">ME</a>
+     
     </div>
     <div class="collapse navbar-collapse" id="myNavbar">
       <ul class="nav navbar-nav navbar-right">
-        <li><a href="#">EDUCATION</a></li>
-        <li><a href="#">WORK EXPERIENCE</a></li>
-        <li><a href="#">CONTACT</a></li>
+        <li><a href="http://127.0.0.1:5501/education.html#">EDUCATION</a></li>
```

Comment:
you hardcoded the URL here. This should direct to local file

Reply:
💯